src/IIIF/IIIF/Utils/CollectionX.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace IIIF.Utils
+{
+    internal static class CollectionX
+    {
+        public static bool IsNullOrEmpty<T>(this List<T> collection)

```

Comment:
It may not be in this project but there is a HasItems() extension that does this in other parts of DDS

Reply:
There aren't any of those helper utils (`HasItems`, `HasText` etc) in this lib yet. I left as `internal` to avoid collisions when this lib is used in other projects